app/xml/XsensSuitWearableDevice.xml

```diff
@@ -18,16 +18,18 @@
         <!-- NposeWalk - TposeWalk - Npose - Tpose-->
         <param name="default-calibration-type">Npose</param>
         <!-- Minimum calibration quality to be considered good enought to be applied and allow the acquisition to start. Available values are: -->
-        <!-- Poor - Acceptable -- Good -->
-        <param name="minimum-calibration-quality-required">Acceptable</param>
+        <!-- Poor - Acceptable - Good -->
+        <param name="minimum-calibration-quality-required">Poor</param>
         <!-- Maximum Time [s] to scan for the suit. Set to -1 enables endless scan-->
         <param name="scan-timeout">60</param>
         <!-- Sampling rate [Hz]. Available values are:-->
         <!-- 240 - 120 - 80 - 60 -->
         <param name="sampling-rate">120</param>
         <!-- Flag to save mvn recording -->
         <param name="saveMVNRecording">false</param>
-        <!-- Quantities to be extracted from the driver for each time sample -->
+		<!-- Flag to save .mvn calibration -->
+		<param name="saveCurrentCalibration">true</param>
+		<!-- Quantities to be extracted from the driver for each time sample -->
```

Comment:
nitpick, this block seems to have a different indentation wrt the rest of the file

Reply:
Done in - Updated `Xsens` wearable device with `saveCurrentCalibration` option. (https://github.com/robotology/wearables/pull/120)

Reply:
Done in https://github.com/robotology/wearables/pull/120/commits/171fb42047f774be4cc4766415817fea218675e7